src/utils/classObserver.ts

```diff
@@ -1,11 +1,14 @@
 export type ObserverIdentifier = number;
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type CallbackFunction = (...args: any[]) => void;
+
 /**
  * The interface to implement if one wants to use a subscriber pattern on the implementing class.
  */
 export interface ObserverInterface {
     id: ObserverIdentifier;
-    callback: Function;
```

Comment:
Needed to change after lint version updates:
![image](https://user-images.githubusercontent.com/13872908/134325134-5fe1bb75-5c3d-4de3-bfdc-1accfb739b8f.png)